evadb/third_party/databases/mysql/mysql_handler.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# Copyright 2018-2023 EvaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import mysql.connector
```

Comment:
Do we need a requirements.txt for `mysql`?

Reply:
yes, This module does not come built-in with Python

Reply:
Thanks for the confirmation. You can add a requirements.txt in the same directory. EvaDB then can automatically pick it up and install the mysql dependency when a connection to mysql is created.

Reply:
my bad, I thought I already added it. looks like Txt files are by default ignored. Adding it now